Technical-Steering-Committee/issues.md

```diff
@@ -0,0 +1,49 @@
+# Issues
```

Comment:
Just to double check. Will the teams have to follow these guidelines to the letter or is there some flexibility? For example, can a team have its own additional labels?

Reply:
We discussed this on last week's TSC, and the intention is to be flexible. The TSC's key goal is for squads to triage issues in the backlog and assign labels to them which can be used to organize them. This document provides a labeling system we think is a good baseline, but its not required to adopt it as-is (or be limited by it).

Reply:
The answer is yes. I will add that the teams can modify and/or add the things to this policy in their main repository in ISSUES.md file